src/storages/types.ts

```diff
@@ -367,6 +381,10 @@ export interface IUniqueKeysCacheBase {
   clear(): void
 }
 
+export interface IUniqueKeysCacheInRedis extends IUniqueKeysCacheBase {
```

Comment:
Same here. Check if you can remove this interface too, since `UniqueKeysCacheInRedis` extends `UniqueKeysCacheInMemory`